libraries/AP_HAL_SITL/HAL_SITL_Class.cpp

```diff
@@ -272,6 +272,7 @@ void HAL_SITL::run(int argc, char * const argv[], Callbacks* callbacks) const
         if (fill_count++ % 10 == 0) {
             // only fill every 10 loops. This still gives us a lot of
             // protection, but saves a lot of CPU
+            fill_count = 1u;
```

Comment:
Why not `= 0;` and change the loop to be `>= 10`?  It's a simpler and faster construct.

Reply:
Thanks for the comment.
"= 0" will be TRUE in the next decision. Therefore, "=1" is used.

I agree.
Change the condition to ">=10" and set the initial value to "=0".

Reply:
In the original process, when the fill_count value is 0, 10, 20, ..., the fill_stack_nan method is executed.
Thus, I do not change it.